src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/NextReviewDateGetterService.kt

```diff
@@ -42,7 +42,7 @@ class NextReviewDateGetterService(
     val existingNextReviewDates = nextReviewDateRepository.findAllById(bookingIds).toList().toMapByBookingId()
 
     val bookingIdsWithDate = existingNextReviewDates.keys
-    val bookingIdsWithoutDate = bookingIds.subtract(bookingIdsWithDate)
+    val bookingIdsWithoutDate = bookingIds subtract bookingIdsWithDate
```

Comment:
😱 not sure whether that's more readable, we may well use `-` on the two sets if that's a thing, although the minus may be potentially confusing.

Reply:
well it's an `infix fun` and that's how it's was previously used
https://github.com/ministryofjustice/hmpps-incentives-api/commit/c8e03d265670f8c87b3fcedab2605397b249c4f1